xclim/atmos/_precip.py

```diff
@@ -48,7 +49,8 @@ class PrTas(Indicator2D):
                                     compute=indices.max_n_day_precipitation_amount,
                                     )
 
-wetdays = Pr(identifier='r{thresh}mm',
+wetdays = Pr(identifier='wetdays',
+             var_name='r{thresh}mm',
              units='days',
              standard_name='number_of_days_with_lwe_thickness_of_precipitation_amount_at_or_above_threshold',
              long_name='Number of Wet Days (precip >= {thresh} mm)',
```

Comment:
same idea here ... this is giving 'Number of Wet Days (precip >= **10 mm/daymm**)